docs/internals/security.txt

```diff
@@ -38,6 +38,41 @@ action to be taken, you may receive further followup emails.
 
 .. _our public Trac instance: https://code.djangoproject.com/query
 
+.. _security-report-evaluation:
+
+How does Django evaluate a report
+=================================
+
+These are criteria used by the security team when evaluating whether a report
+requires a security release:
+
+* The vulnerability is within a :ref:`supported version <security-support>` of
+  Django.
```

Comment:
Should be add here the other two common false-positive reports about:
1. issues in djangoproject.com (particularly that the admin is publicly accessible at `/admin/`)
2. the DMARC thingy? "we do not consider the lack of SPF or DMARC record an issue."

Reply:
As these are specific to djangoproject.com, so I don't think it fits in this specific list because I'm things that require a security release in Django core
But I do think we should document this somewhere. Maybe within the README of djangoproject.com :thinking: